packages/webviz-core/src/util/debouncePromise.test.js

```diff
@@ -9,6 +9,11 @@
 import debouncePromise from "./debouncePromise";
 import signal from "webviz-core/shared/signal";
 
+process.on("unhandledRejection", () => {
+  // Have a dummy handler for all unhandled rejections
+  // This is to prevent UnhandledPromiseRejection on promises that we reject in tests here
+});
+
```

Comment:
I wonder if we should put this somewhere more generic like `webviz-core/src/test/setup.js`?

Reply:
I think it's fine to leave as is - we only have 2 places where we use it and I would rather have less indirection for 2 lines of code (if 5 or more then it would make more sense to extract in my opinion)